ExampleMod/Content/Achievements/ManyExampleWormsKilled.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ExampleMod.Content.Items.Placeable;
+using Terraria;
+using Terraria.Achievements;
+using Terraria.GameContent.Achievements;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.Achievements;
+
+public class ManyExampleWormsKilled : ModAchievement
+{
+	public override void SetStaticDefaults() {
+		Achievement.Hidden = true;
+
+		// Int conditions will automatically complete once you've incremented it enough.
+		// ((CustomIntCondition)ModContent.GetModAchievement<ManyExampleWormsKilled>().GetCondition("EXAMPLEMOD_KILL_WORMS")).Value++;
+		Achievement.AddCondition(CustomIntCondition.Create("EXAMPLEMOD_KILL_WORMS", 5));
+		Achievement.UseTrackerFromCondition("EXAMPLEMOD_KILL_WORMS");
```

Comment:
As with the advice above, recommend making the `Condition` a field, and then using `UseTracker(Condition.GetAchievementTracker())`